site/content/docs/5.0/components/modal.md

```diff
@@ -577,7 +577,7 @@ If the height of a modal changes while it is open, you should call `myModal.hand
 
 ### Accessibility
 
-Be sure to add `role="dialog"` and `aria-labelledby="..."`, referencing the modal title, to `.modal`. Additionally, you may give a description of your modal dialog with `aria-describedby` on `.modal`.
+Be sure to add `aria-labelledby="..."`, referencing the modal title, to `.modal`. Additionally, you may give a description of your modal dialog with `aria-describedby` on `.modal`.
```

Comment:
@rohit2sharma95 can you reword this so that we mention that we already add `role="dialog" ourselves?

Reply:
> Be sure to add `aria-labelledby="..."`, referencing the modal title, to `.modal`. You don't need to add `role="dialog"`, we already add it ourselves via script. Additionally, you may give a description of your modal dialog with `aria-describedby` on `.modal`.

Should I change it as above  @XhmikosR?

Reply:
> Be sure to add `aria-labelledby="..."`, referencing the modal title, to `.modal`. Additionally, you may give a description of your modal dialog with `aria-describedby` on `.modal`. Note that you don't need to add `role="dialog"` since we already add it via JavaScript.

Reply:
@XhmikosR 
Updated. Thanks for your suggestion.